Sources/NIOHTTP1/NIOHTTPObjectAggregator.swift

```diff
@@ -397,3 +401,8 @@ public final class NIOHTTPClientResponseAggregator: ChannelInboundHandler, Remov
         self.buffer = context.channel.allocator.buffer(capacity: 0)
     }
 }
+
+
+#if swift(>=5.5) && canImport(_Concurrency)
+extension NIOHTTPClientResponseAggregator: @unchecked Sendable {}
```

Comment:
There's no way this type is `Sendable`.

Reply:
Ups 🫣... fixed in https://github.com/apple/swift-nio/pull/2213/commits/9d78041cee755bf0870aac569bb583545d0e8321